src/App.css

```diff
@@ -1,28 +1,179 @@
-.app {
-  height: 100vh;
-  width: 100vw;
-  transition: background-color 1s ease-in;
+.app__header {
+  background-color: dodgerblue;
+  color: white;
+  display: flex;
+  height: 80px;
 }
-
-.app__header, 
-.app__main {
+.currentWeather,
+.Forcast {
+  height: 100%;
   width: 100%;
+  transition: background-color 1s ease-in;
+  background-color: lightskyblue;
+  font-family: "Raleway";
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 20px;
-  font-family: 'Raleway', sans-serif;
+  font-family: "Raleway", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-
 /*  Search bar */
 
```

Comment:
The SearchBar doesn't have a font-family - can you add it so it looks the same as the rest of the app?